pkg/kubelet/cm/memorymanager/policy_static.go

```diff
@@ -185,7 +185,7 @@ func (p *staticPolicy) Allocate(s state.State, pod *v1.Pod, container *v1.Contai
 
 // RemoveContainer call is idempotent
 func (p *staticPolicy) RemoveContainer(s state.State, podUID string, containerName string) error {
-	klog.Infof("[memorymanager] RemoveContainer (pod: %s, container: %s)", podUID, containerName)
+	klog.InfoS("[memorymanager] RemoveContainer", "podUID", podUID, "container", containerName)
```

Comment:
```suggestion
	klog.InfoS("[memorymanager] RemoveContainer", "podUID", podUID, "containerName", containerName)
```

Reply:
at all places below as well, we are keeping consistency across key names as we are printing `container.Name` so the key would be `containerName`. `container` is the key for container object.